src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -26,6 +26,7 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
             keyboardAppearDelay={1}
             editable={!props.isOtherProfile}
             maxLength={350}
+            placeholderTextColor={COLORS.gray400}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The placeholder text color has been changed from the default to `COLORS.gray400`. This change should improve readability and user experience by providing a more visually appealing contrast between the placeholder text and the background.
```diff
- placeholderTextColor={defaultColor}
+ placeholderTextColor={COLORS.gray400}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->